OpenRA.Mods.Common/Widgets/Logic/Installation/InstallFromDiscLogic.cs

```diff
@@ -382,7 +383,7 @@ enum ExtractionType { Raw, Blast }
 		static void ExtractFromPackage(ModData modData, ExtractionType type, string path, MiniYaml actionYaml, List<string> extractedFiles, Action<string> updateMessage)
 		{
 			// Yaml path may be specified relative to a named directory (e.g. ^SupportDir) or the detected disc path
-			var sourcePath = actionYaml.Value.StartsWith("^") ? Platform.ResolvePath(actionYaml.Value) : Path.Combine(path, actionYaml.Value);
+			var sourcePath = FS.ResolveCaseInsensitivePath(actionYaml.Value.StartsWith("^") ? Platform.ResolvePath(actionYaml.Value) : Path.Combine(path, actionYaml.Value));
```

Comment:
Named OpenRA directories already have a well defined case, so these should be `var sourcePath = actionYaml.Value.StartsWith("^") ? Platform.ResolvePath(actionYaml.Value) : FS.ResolveCaseInsensitivePath(Path.Combine(path, actionYaml.Value));` IMO.

(sorry, I didn't notice this on the last pass)